cli/src/template/init/build.rs

```diff
@@ -1,10 +1,15 @@
+use std::path::Path;
+
 fn main() {
     // CIRCOM_TEMPLATE
     // This is writing the UDL file which defines the functions exposed
-    // to your app. We have pre-generated this file for you
-    // This file must be written to ./src
-    std::fs::write("./src/mopro.udl", mopro_ffi::app_config::UDL).expect("Failed to write UDL");
+    // to your app. We have pre-generated this file for you.
+    // Feel free to modify it to suit your needs.
+    let udl_path = Path::new("src/mopro.udl");
+    if !udl_path.exists() {
+        std::fs::write(udl_path, mopro_ffi::app_config::UDL).expect("Failed to write UDL");
+    }
     // Finally initialize uniffi and build the scaffolding into the
     // rust binary
-    uniffi::generate_scaffolding("./src/mopro.udl").unwrap();
+    uniffi::generate_scaffolding(udl_path).unwrap();
```

Comment:
it should be a string type
e.g. `udl_path.to_str().unwrap()`

Reply:
I think CLI needs some tests that would compile these files, should I create an issue for that?

Reply:
sounds good to me!
and anything you think we can integrate with `clippy` you can also create an issue. 🙏🏻